src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawable {
```

Comment:
Separate the concerns of calculating area and drawing into two distinct interfaces, as they are conceptually different. One interface should be responsible for calculating the area, and another for drawing.